packages/strapi-admin/admin/src/components/Roles/Permissions/ContentTypes/ContentTypesRow/ContentTypesAttributes/AttributeRow.js

```diff
@@ -76,17 +74,19 @@ const AttributeRow = ({ attribute, contentType }) => {
       const allActionsSize = attributes.length * ATTRIBUTES_PERMISSIONS_ACTIONS.length;
       const shouldEnable = recursivePermissions >= 0 && recursivePermissions < allActionsSize;
 
-      onAllContentTypeActions({
+      dispatch({
+        type: 'ALL_CONTENT_TYPE_PERMISSIONS_SELECT',
```

Comment:
Also, you don't need the `shouldEnable` constant and the `contentType.uid` you can use the target

```
const handleCheckAllActions = ({ target: {  name, value }) => {
  // In order to have the name matches the content type uid you need to change the name of the checkbox to contentType.uid
  dispatch({ type, attributes, shouldEnable: value, subject: name})
}
```